### DISCUSSION TITLE
Using Current Accumulative Cost in the energy dashboard vs Current Rate

### DISCUSSION THREAD

**Discussion by addrum** [BottlecapDave/HomeAssistant-OctopusEnergy#368] [Q&A] [answered]
Just a QQ; how come the [docs](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/energy_dashboard.md#for-electricity) suggest to use the current rate which won't calculate the standing charge whereas using the Current Accumulative Cost _does_ include the standing charge? To have HA more accurately track the cost wouldn't it be better to use the figure that includes the standing charge?

I have found [this thread](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/330) which suggests using a template to include the standing charge and then add to the energy dashboard but that _seems_ redundant to me?

Also found [this one](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/355) discussing a possibly related bug where it looks like the poster _is_ using Current Accumulative Cost.

I'm currently using the rate setup but just seeing if I'm missing anything as I'd prefer to have the standing charge included!
Thanks!

**Comment by BottlecapDave** [comment, marked as answer]
Hi there.

Long story short - The current accumulative sensors were new in the latest release and I haven't updated the docs yet :)

The first thread you linked was in relation to wanting peak/off peak breakdowns in the energy dashboard. These sensors do not include the standing charge and so this was why a template sensor was recommended so that the combined cost of peak/off peak included the standing charge.

The second issue will be resolved in the next release that I'm currently just testing.
